docs/reference-guide/temporal_types/overview.md

```diff
@@ -0,0 +1,238 @@
+---
+id: overview
+title: Temporal types
+sidebar_label: Temporal types overview
+slug: /reference-guide/temporal_types/
+---
+
+Temporal types can be stored as properties of a graph. The user can use cypher queries or write/read procedures to write/read these properties. The following table summarizes the temporal types that memgraph currently supports:
+
+name|description|
+:-:|:-:
+Duration|Data type that represents a duration.|/
+Date|Data type that represents a date.|/
+LocalTime|Data type that represents local time.|/
+LocalDateTime|Data type that represents a date and local time.|/
+
+Note, that all the types excluding Durations are ISO 8601 compliant
+
+## Duration
+You can create `Duration` from a string or a map by calling the function `Duration`.
+
+For strings, the duration format is:
+
+`P[nD]T[nH][nM][nS]` n stands for number which can be a fraction  and the capital letters are used as separator with each field in `[]` being optional. For strings, Memgraph only allows one field to be of type double. However, for maps, every field can be a double, an int or a mixture of both. Note, we support negative durations.
+
+Symbol table
```

Comment:
I would either convert this to a sub-heading or remove it. The same true for `Duration(/date/locatime/localdatetime) properties`